save-demo/src/main/kotlin/com/saveourtool/save/demo/SaveDemo.kt

```diff
@@ -0,0 +1,16 @@
+package com.saveourtool.save.demo
+
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
+import org.springframework.boot.autoconfigure.jdbc.XADataSourceAutoConfiguration
+
+/**
+ * An entrypoint for spring boot for save-demo
+ */
+@SpringBootApplication(exclude = [DataSourceAutoConfiguration::class, XADataSourceAutoConfiguration::class])
```

Comment:
I think you could easily remove this, since you have no JPA dependency

Reply:
Seems like some dependencies (probably `libs.save.common.jvm`) still carry JPA dependencies so I still should mark those configurations as excluded

Reply:
It may be `projects.saveCloudCommon`, because `save.common` is part of `save-cli`. Anyway, it's strange that we use a dependency with autoconfiguration classes there